pkg/sql/logictest/testdata/planner_test/show_trace

```diff
@@ -134,7 +134,7 @@ statement ok
 SET tracing = on,kv,results; UPDATE t.kv2 SET v = v + 2; SET tracing = off
 
 query TT
-SELECT operation, regexp_replace(message, '(\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.)?\d\d\d\d\d+', '...PK...') as message
```

Comment:
Using the row converter changes the way row_id's are generated for hidden PK's. They are generated using `GenerateUniqueID` which required this change in the logic test.